index.html

```diff
@@ -46,43 +44,37 @@
                     </svg>
                 </a>
             </div>
-        </div>
+        </nav>
     </header>
 
     <!-- Fly with us section -->
     <section class="fly-with-us">
 
         <div class="container">
-            <h2 class="content-title">
-                <span class="underline">Fly with us to</span>
+            <h2 class="fly-with-us__content-title">
+                <span class="fly-with-us__underline">Fly with us to</span>
             </h2>
             <div class="tabs">
-                <div class="tab">
-                    <h3 class="tab__name">01</h3>
-                    <h3 class="tab__dot">.</h3>
-                </div>
-                <div class="tab">
-                    <h3 class="tab__name">02</h3>
-                    <h3 class="tab__dot">.</h3>
-                </div>
-                <div class="tab">
-                    <h3 class="tab__name">03</h3>
-                    <h3 class="tab__dot">.</h3>
-                </div>
+                <input class="tab__input" type="radio" id="tab-1" name="tab" value="1">
```

Comment:
I was not expecting radio buttons, cool you could make slider with just CSS :) I just meant that you use the `::after` for dots, which you did 👍